src/main/java/com/enterprise/fundraisermanager/dao/ITransactionDAO.java

```diff
@@ -1,5 +1,15 @@
 package com.enterprise.fundraisermanager.dao;
 
+import com.enterprise.fundraisermanager.dto.Transaction;
+
+import java.util.List;
+
 public interface ITransactionDAO {
+    Transaction save(Transaction transaction) throws Exception;
+
+    List<Transaction> fetchAll();
+
+    Transaction fetch(int transactionId);
 
+    void delete(int transactionId);
 }
```

Comment:
This interface was empty so I added a few a few functions that I thought might be needed.